suites/test_view_assembly.py

```diff
@@ -103,15 +95,14 @@ def preview_assembly(self):
             try:
                 # subprocess.check_call(cmd, shell=True)
                 path = subprocess.getoutput(cmd)
-                print(path)
+                print("Image file path::", path)
+                image_file = "/content/repositories/" + test_repo_name + "/entities/enterprise/assemblies/images/" + constants.image_file_name
+                check_that("Path to image1", path, equal_to(image_file))
             except subprocess.CalledProcessError as e:
                 logging.error("Unable to decode imageasset")
```

Comment:
I would suggest lcc.log_error so that it shows up in the report along with the error string: str(e).